libraries/chain/include/eosio/chain/fork_database.hpp

```diff
@@ -12,6 +12,7 @@ namespace eosio::chain {
    using block_branch_t = std::vector<signed_block_ptr>;
    enum class ignore_duplicate_t { no, yes };
    enum class include_root_t { no, yes };
+   enum class fork_db_add_t { failure, duplicate, added, appended_to_head, fork_switch };
```

Comment:
Should comments below (lines 71 to 74) be moved here?

Reply:
https://github.com/AntelopeIO/spring/pull/1047/commits/5e16cd5e29076e72ef804c083bc95c5ea7ba6a51